roles/teams/tasks/main.yml

```diff
@@ -57,7 +56,6 @@
   changed_when: "gsh_output.stderr != ''"
   args:
     chdir: /opt/www/grouper-shell
-  when: env == 'vm'
```

Comment:
The script itself checks whether the stem already exists, and if so, does nothing.  Also, creating additional stems, if it would happend by accident, doesn't matter for Grouper or Temas operations.  A missing stem, otoh, does matter quite a lot.